### PR TITLE
MARKET-2301 [Support - Tabs] Improve work with tabs.

### PR DESCRIPTION
**Pull request by StasivPavlo**

### CODE REVIEW

**Comment by StasivPavlo** [comment]
I think changing the usage guide's last point will be better. It is used there a `getCurrentId` action to change the visibility that doesn't retrigger when the `currentTabId` changes. For example, when I determine the current tab id by logic handler, tabs aren't visible, but when I change the action to the "Tabs Content" data model in visibility logic it works. What do you think about it?

**Review by StasivPavlo — COMMENTED** [review]
I think changing the usage guide's last point will be better. It is used there a getCurrentId action to change the visibility that doesn't retrigger when the currentTabId changes. For example, when I determine the current tab id by logic handler, tabs aren't visible, but when I change the action to the "Tabs Content" data model in visibility logic it works. What do you think about it?

**Comment by v-excelsior** [comment]
> when I determine the current tab id by logic handler, tabs aren't visible

It's bug, and it should be fixed. Isn't it?

File: components/bl-tabs/README.md [hunk 1]
```diff
@@ -12,18 +12,21 @@ View an example of how to install this component and how it works in your UI [he
...
 ## Events
 
-| Name                  | Triggers                        | Context Blocks               |
-|-----------------------|---------------------------------|------------------------------|
-| On Change Event       | triggered when a tab is changed | Current Tab Id: `String`     |
+| Name              | Triggers                                    | Context Blocks               |
+|-------------------|---------------------------------------------|------------------------------|
+| On Change Event   | triggered when a tab is changed             | Current Tab Id: `String`     |
+| On Mounted        | triggered after the component is mounted    |                              |
+| On Before Unmount | triggered before the component is unmounted |                              |
```
Comment (serhmel) [inline components/bl-tabs/README.md:29]:
```suggestion
| On Mounted Event        | triggered after the component is mounted    |                              |
| On Before Unmount Event | triggered before the component is unmounted |                              |
```
?

File: components/bl-tabs/src/index.js [hunk 2]
```diff
@@ -47,10 +47,18 @@ export default function TabsComponent({ component, eventHandlers, pods }) {
     }
   }, [currentTabId, tabsList]);
 
+  useEffect(() => setCurrentTabId(currentTab), [currentTab]);
+
   useEffect(() => {
     podsContent.dataModel.currentTabId = currentTabId;
```
Comment (serhmel) [inline components/bl-tabs/src/index.js:53]: I do not fully understand where we use `currentTabId` from `dataModel`.
I assume that @StasivPavlo is right, and it should have been used in Visibility Logic.
But taking into account that we add `Current Tab Id`  property, I assume that we could use it instead.

Comment (v-excelsior) [reply components/bl-tabs/src/index.js:53]: Now it runs on unmount too

File: components/bl-tabs/component.json [hunk 3]
```diff
@@ -82,6 +82,19 @@
...
+      "name": "currentTab",
+      "label": "Current Tab Id",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "currentTabLogic",
+      "handlerLabel": "Current Tab Id Logic",
+      "dataBinding": true,
+      "handlerDescription": "This handler allows you to determine the current tab.",
+      "defaultValue": "tab1",
+      "settingTooltip": "This handler allows you to determine the current tab."
```
Comment (serhmel) [inline components/bl-tabs/component.json:97]: Don't use handler description for property tooltip.

**Pull request merged**